lib/plug/templates/mailbox_viewer/index.html.eex

```diff
@@ -4,203 +4,159 @@
+              <div class="flex-grow">&nbsp;</div>
+              <div class="flex items-center justify-center pb-2">
+                <form method="POST" action="<%= to_absolute_url(@conn, "clear") %>">
+                  <input name="_csrf_token" type="hidden" value="<%= Plug.CSRFProtection.get_csrf_token() %>">
+                  <button class="bg-blue-500 hover:bg-blue-400 text-white text-sm w-64 py-3 px-4 rounded" type="submit">Empty mailbox</button>
+                </form>
+              </div>
+            </div>
+          <% end %>
         </div>
+      </div>
 
-        <div class="col-sm-3 offset-sm-3 col-sm-9 content">
-          <%= if @email do %>
-            <div class="header">
-              <div class="header-content">
-                <dl class="dl-horizontal headers">
-                  <dt class="col-sm-2">Subject</dt>
-                  <dd class="col-sm-10">
-                    <%= if @email.subject == "" do %>
-                      <i>No subject</i>
-                    <% else %>
-                      <%= @email.subject %>
-                    <% end %>
-                  </dd>
-
-                  <dt class="col-sm-2">From</dt>
-                  <dd class="col-sm-10"><%= render_recipient(@email.from) %></dd>
-
-                  <dt class="col-sm-2">To</dt>
-                  <dd class="col-sm-10"><%= render_recipient(@email.to) %></dd>
-
-                  <dt class="col-sm-2">Cc</dt>
-                  <dd class="col-sm-10"><%= render_recipient(@email.cc) %></dd>
-
-                  <dt class="col-sm-2">Bcc</dt>
-                  <dd class="col-sm-10"><%= render_recipient(@email.bcc) %></dd>
-
-                  <dt class="col-sm-2">Reply-to</dt>
-                  <dd class="col-sm-10"><%= render_recipient(@email.reply_to) %></dd>
+      <div class="w-full">
+        <%= if @email do %>
+          <div class="">
+            <div class="">
+              <div class="">
+                <dl class="divide-y divide-gray-100">
+                  <div class="p-4">
+                    <dt class="text-sm font-medium leading-6 text-gray-900">From</dt>
+                    <dd class="mt-1 text-sm leading-6 text-gray-700"><%= render_recipient(@email.from) %></dd>
+                  </div>
+
+                  <div class="p-4">
+                    <dt class="text-sm font-medium leading-6 text-gray-900">To</dt>
+                    <dd class="mt-1 text-sm leading-6 text-gray-700"><%= render_recipient(@email.to) %></dd>
+                  </div>
+
+                  <div class="p-4">
+                    <dt class="text-sm font-medium leading-6 text-gray-900">Subject</dt>
+                    <dd class="mt-1 text-sm leading-6 text-gray-700">
+                      <%= if @email.subject == "" do %>
+                        <i>No subject</i>
+                      <% else %>
+                        <%= @email.subject %>
+                      <% end %>
+                    </dd>
+                  </div>
```

Comment:
They (including the next few chunks) are taking quite a bit of vertical space. On smaller screens they'd require the viewer to scroll to see the body. Do you think they'd be better turned into a 2-column table?

Reply:
I tried that at first but bailed for 2 reasons, 1. the eye movement necessary to just quickly scan if all info was in the right place and 2. it didn't look as good. I can try to tighten it up a bit, I know there's a lot of "air" in the layout but that is to make it easier to glance information.